flink-formats/flink-json/src/test/java/org/apache/flink/formats/json/JsonRowDataSerDeSchemaTest.java

```diff
@@ -330,12 +328,12 @@ public void testSerDeMultiRows() throws Exception {
             byte[] serializedJson = objectMapper.writeValueAsBytes(root);
             RowData rowData = deserializationSchema.deserialize(serializedJson);
             byte[] actual = serializationSchema.serialize(rowData);
-            assertEquals(new String(serializedJson), new String(actual));
+            assertThat(actual).isIn(serializedJson);
```

Comment:
this is not correct. It allows serializedJson to contain more stuff. You should use containsExactly. Also, let's switch the assertion order.